### ISSUE TITLE
Add option for `tab` or `enter` geocode completion

### ISSUE THREAD

**Issue by amyleew** [perliedman/leaflet-control-geocoder#88]
It'd be nice to have an option for geocoder completion on `tab` in addition to `enter` 

Current user interaction:

 - Type address in first geocoder
 - Tab to next geocoder, type address
 - Hit enter. (*Nothing happens*)
 - Hit enter again (*Only end geocoder gets geocoded/pin drop*)

**Comment by perliedman** [comment]
I think I totally agree with this. Just to be sure, the interaction you describe above relates to [Leaflet Routing Machine]((https://github.com/perliedman/leaflet-routing-machine), right? Although totally not obvious, the UI for the geocoders in LRM is actually implemented in LRM - it's just leaflet-control-geocoder's communication code for performing requests and parsing the results that is used in LRM.

I went ahead and created a new issue on LRM: [#148](https://github.com/perliedman/leaflet-routing-machine/issues/148)

Keeping this open, as it might actually also be a good idea for this plugin as well!

**Comment by amyleew** [comment]
@perliedman Yes. Trying to implement in a project using LRM. Searching the code for a workaround lead to the believe it originated from this plugin. Thanks for adding the issue to LRM as well!
